Source/Widgets/Animation/Animation.js

```diff
@@ -351,6 +351,8 @@ define([
         this._lastHeight = undefined;
         this._lastWidth = undefined;
 
+        var doc = container.ownerDocument;
```

Comment:
Sorry to be picky, but can we call this variable `ownerDocument` just so its crystal clear to the reader that `doc !== document`?

Reply:
Same comment throughout, just want to avoid future confusion and silly mistakes.